### DISCUSSION TITLE
dd() on chrome not working.

### DISCUSSION THREAD

**Discussion by adityadees** [laravel/framework#50785] [Q&A]
I frequently use Google Chrome as my browser, but I'm starting to feel a bit frustrated because the output from dd() function will be encoding characters. Interestingly, Firefox doesn't seem to have this issue; everything works smoothly without character encoding. Should this issue be addressed by @laravel or by  @googlechrome ?

Firefox
<img width="479" alt="image" src="https://github.com/laravel/framework/assets/37553901/7181254a-39ee-4d56-9f92-0ef31a7cdd7e">

Chrome
<img width="290" alt="image" src="https://github.com/laravel/framework/assets/37553901/398617c3-7a17-4f50-ae3e-831af7bc5d3e">

**Comment by rojtjo** [comment]
I don't seem to have this issue in Chrome. Do you happen to be running any extensions that might cause this?

![image](https://github.com/laravel/framework/assets/1123887/1c00865d-a936-4781-bfb0-db551c41ba0a)

Also, the `dd()` and `dump()` functions are from [symfony/var-dumper](https://github.com/symfony/var-dumper) package.

**Reply by adityadees** [reply]
try to use latest version of chrome I think @rojtjo

**Reply by rojtjo** [reply]
Yeah I realized I was still on v122 after reading the other comment!

**Comment by mbabker** [comment]
This is a Chrome bug as confirmed in https://github.com/symfony/symfony/issues/54397

**Reply by adityadees** [reply]
yeah then waiting chrome to update I think, now will move to firefox instead
